### PR TITLE
Adding Cisco CVE-2022-20680

### PR DESCRIPTION
**Pull request by santosomar**

### CODE REVIEW

File: 2022/CVE-2022-20630.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
```
Comment (cve-team) [inline 2022/CVE-2022-20630.json:1]: Expected location (2022/20xxx/CVE-2022-20630.json) of file does not match actual location (2022/CVE-2022-20630.json)

File: 2022/CVE-2022-20680.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
+{
```
Comment (cve-team) [inline 2022/CVE-2022-20680.json:1]: Expected location (2022/20xxx/CVE-2022-20680.json) of file does not match actual location (2022/CVE-2022-20680.json)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hello @santosomar,

All 17 PR requests are waiting for you to address the validation errors. Once cleared up we can then process and merge these records. 

Thanks,
CVE Content Team

**Comment by anthnysingleton** [comment]
This PR has been opened with no response from submitter for 7 days. MITRE CVE Content team has manually processed the file in order to get the information to the main list and its down stream users. 

Thanks,
CVE Content Team

**Pull request closed without merging**